### PR TITLE
Add icon set page to the style guide

### PR DESCRIPTION
**Pull request by marzetas**
Add link to it in the frontend section
- [x] @io41 
- [x] @full-of-foo 
- [x] @Xevi 
- [x] @alvaroortega 
- [x] @basil-conto

### CODE REVIEW

**Comment by io41** [comment]
Haven't tested the pages but looks sane. :+1:

File: icomoon/demo-files/demo.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+document.body.addEventListener("click", function(e) {
+	var target = e.target;
+	if (target.tagName === "INPUT" &&
+		target.getAttribute('class').indexOf('liga') === -1) {
+		target.select();
+	}
+});
+
+(function() {
+	var fontSize = document.getElementById('fontSize'),
```
Comment (alvaroortega) [inline icomoon/demo-files/demo.js:14]: I agree with the "best practices We disagree with" section in this case. This way to declare variables could cause problems if we miss a coma for instance.

I would declare three different vars

Comment (marzetas) [inline icomoon/demo-files/demo.js:14]: I'd prefer not to touch any of the files under the icomoon folder, as those files are generated here: https://icomoon.io/

Comment (alvaroortega) [inline icomoon/demo-files/demo.js:14]: Ah, sorry I thought all files were part of the review. +1 in that case :smile:

**Pull request merged**